datascience-notebook/Dockerfile

```diff
@@ -11,9 +11,16 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     fonts-dejavu \
     gfortran \
+    unixodbc-dev \
+    libtool-bin \
+    autoconf \
+    automake \
     gcc && apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+# Python pyodbc
+RUN pip install pyodbc
```

Comment:
When I run conda install for pyodbc I get v3.0.10. I get the same when running pip install. I'm assuming their equivalent.

Re: database to test against, I think that's something we're going to want to document in the README (see below). Linking to one of the official maria, mysql, postgres, etc. docker containers to do it is probably the simplest way.